powersimdata/input/input_data.py

```diff
@@ -3,139 +3,46 @@
 
 import pandas as pd
 
-from powersimdata.data_access.context import Context
-from powersimdata.data_access.profile_helper import ProfileHelper
+from powersimdata.input.input_base import InputBase
 from powersimdata.utility import server_setup
-from powersimdata.utility.helpers import MemoryCache, cache_key
 
-_cache = MemoryCache()
 
-profile_kind = {
-    "demand",
-    "hydro",
-    "solar",
-    "wind",
-    "demand_flexibility_up",
-    "demand_flexibility_dn",
-    "demand_flexibility_cost_up",
-    "demand_flexibility_cost_dn",
-}
-
-
-_file_extension = {
-    **{"ct": "pkl", "grid": "mat"},
-    **{k: "csv" for k in profile_kind},
-}
+class InputData(InputBase):
+    """Load input data."""
 
+    def __init__(self):
+        super().__init__()
+        self._file_extension = {"ct": "pkl", "grid": "mat"}
 
-class InputHelper:
-    @staticmethod
-    def get_file_components(scenario_info, field_name):
-        """Get the file name and relative path for either ct or grid.
+    def _get_file_path(self, scenario_info, field_name):
```

Comment:
I think we can still use the staticmethod decorator here

Reply:
Think that would require moving `self._file_extension` outside the class. Feels simpler to have it be an instance method since I'm not 100% sure about static methods and inheritance (though it does seem to work similarly).

Reply:
Oops, I did not see  that the `_file_extension` attribute was used

Reply:
AFAIK, static methods will work with inheritance, as well as class methods, while the latter will be more convenient when dealing with multilayer inheritance.